plugins/modules/managed_serviceaccount_addon.py

```diff
@@ -1,6 +1,15 @@
 #!/usr/bin/python
 # GNU General Public License v3.0+ (see COPYING or https://www.gnu.org/licenses/gpl-3.0.txt)
 from __future__ import (absolute_import, division, print_function)
+from ansible_collections.ocmplus.cm.plugins.module_utils.addon_utils import (
```

Comment:
is this auto formatting? ansible requires import to be after DOCUMENTATION

Reply:
Right, I used `autopep8 --in-place --recursive managed_serviceaccount_addon.py` to do the formatting

Reply:
I fixed the formatting error